From f991d8c7f0d19ccc948ec6540487649adf20d99b Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 22 May 2013 08:45:03 +0200
Subject: Disable unsupported emulated SCSI devices

Bugzilla: 903918

We do not support following device so it has to be disabled:
- lsi53c895a
- am53c974
- dc390
- esp
- megasas
- pvscsi

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>

diff --git a/default-configs/pci.mak b/default-configs/pci.mak
index 78f1579..8ef7e50 100644
--- a/default-configs/pci.mak
+++ b/default-configs/pci.mak
@@ -11,9 +11,6 @@ CONFIG_PCNET_COMMON=y
 CONFIG_AC97=y
 CONFIG_HDA=y
 CONFIG_ES1370=y
-CONFIG_LSI_SCSI_PCI=y
-CONFIG_VMW_PVSCSI_SCSI_PCI=y
-CONFIG_MEGASAS_SCSI_PCI=y
 CONFIG_RTL8139_PCI=y
 CONFIG_E1000_PCI=y
 CONFIG_VMXNET3_PCI=y
@@ -21,8 +18,6 @@ CONFIG_IDE_CORE=y
 CONFIG_IDE_QDEV=y
 CONFIG_IDE_PCI=y
 CONFIG_AHCI=y
-CONFIG_ESP=y
-CONFIG_ESP_PCI=y
 CONFIG_SERIAL=y
 CONFIG_SERIAL_PCI=y
 CONFIG_IPACK=y
